internal/ethereum/get_receipt.go

```diff
@@ -166,20 +166,7 @@ func (c *ethConnector) getErrorMessage(ctx context.Context, transactionHash stri
 		return nil, nil
 	}
 
-	parsedABI, err := abi.ParseABI([]byte(`[
-			{
-				"name": "Error",
-				"inputs": [
-					{"name": "message", "internalType": "string", "type": "string" }
-				],
-				"type": "function"
-			}
-		]`))
-	if err != nil {
-		return nil, err
-	}
-
-	value, err := decodeCallDataHex(ctx, parsedABI.Functions()["Error"], returnValue)
+	value, err := decodeCallDataHex(ctx, defaultError, returnValue)
```

Comment:
We had a duplicate definition of this API (once in string JSON, and the other as a pre-parsed ABI)